portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Santiago's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Santiago's Portfolio</h1>
+      <img id="profile" src="/images/profile.jpg"/>
+      <p>I'm Santiago, a Computer Science student at Tecnológico de Monterrey. Very passionate about everything related to coding, A.I., C.S., and tech in general. Developed Ecoogo (ecoogo.com.co), an awarded app for eco-friendly establishments. I also love math, solving problems, and getting the best out of every team I work with.</p>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to get a random message:</p>
```

Comment:
nit: might make sense to change:
random message -> fun fact about me

Reply:
Yup! I changed it.